apps/bot_manager/lib/bot_state_machine_checker.ex

```diff
@@ -35,36 +47,60 @@ defmodule BotManager.BotStateMachineChecker do
       cap_for_ultimate_skill: 3,
       previous_position: nil,
       current_position: nil,
-      time_to_change_direction: 1600,
-      last_time_direction_changed: 0,
-      current_time_in_direction: 0
+      position_to_move_to: nil,
+      time_amount_to_change_position: 2000,
+      last_time_position_changed: 0,
+      max_vision_range_to_follow_player: 1500,
+      vision_range_to_attack_player: 1200,
+      stuck_in_position: nil,
+      start_time_stuck_in_position: nil
```

Comment:
Let's create an issue to remove these magic numbers to make them parametrizable.

Reply:
Done in #1067